### ISSUE TITLE
[Accessbility]A11y_Java Tooling for IntelliJ/Eclipse - New Feature_HDInsight Library Settings_Screen Reader: Narrator/NVDA is not announcing the button-name "Use as Source Folder" 

### ISSUE THREAD

**Issue by Manisha370** [microsoft/azure-tools-for-java#3572]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse-NewFeature; #MAS1.3.1; #ScreenReader; #C+E-H; #JavatoolingforIntelliJ/Eclipse; #Narrator; #NVDA

**Environment Details:**
Eclipse IDE for Enterprise Java Developers
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0

**OS Details:**
OS: Windows 10 Enterprise (Version 1903, Build: 18362.356)
Screen Reader: Narrator, NVDA
MAS Violated: MAS1.3.1

**Note:**
**Similar issue is observed for the controls "Link additional source to project" button & "Opens the new Java source folder wizard" button within the same page**

**Repro Steps:**
1.Turn on Screen Reader
2.Open Java EE - Eclipse application
3.Install 'Azure Toolkit for Eclipse' from Marketplace
4.Open Eclipse and install  the Scala plug-in 
5.Tab till tools-->Azure-->Azure Sign in and sign into Azure account and select any subscription
6.Tab till File-->New-->Project and press enter
7. Tab till HDInsight and expand it. Select 'Spark on HDInsight (Scala)'. Tab till Next and press enter
8.Enter the Project name & tab till "Next" button and press enter
9.Tab till "Use as Source Folder" button
10.Verify whether screen reader is announcing the label name "Use as Source Folder" or not

**Actual Result:**
Narrator is announcing as button, when focus on "Use as Source Folder" button
NVDA is announcing as tool bar button

**Expected Result:**
Narrator/NVDA should announce as "Use as Source Folder" button 

**User Impact:**
Screen Reader user will not be able to differentiate the buttons, if button names are not read by them

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

[Bug5_Narrator is not announcing the button name.pptx](https://github.com/microsoft/azure-tools-for-java/files/3635935/Bug5_Narrator.is.not.announcing.the.button.name.pptx)
![Bug5_NVDA is not announcing the button name](https://user-images.githubusercontent.com/51435937/65327791-7f7ca100-dbd2-11e9-8613-b032c38567e7.png)

**Comment by t-rufang** [comment]
It is determined by Eclipse IDE. We cannot fix it. And it's out of the toolkit's scope.

**Comment by Alexandria-R** [comment]
This has been approved to close by SME.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749738
